src/quo2/components/list_items/saved_contact_address/view.cljs

```diff
@@ -0,0 +1,94 @@
+(ns quo2.components.list-items.saved-contact-address.view
+  (:require [quo2.components.avatars.user-avatar.view :as user-avatar]
+            [quo2.components.avatars.account-avatar.view :as account-avatar]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.list-items.saved-contact-address.style :as style]
+            [reagent.core :as reagent]
+            [quo2.components.icon :as icon]
+            [utils.i18n :as i18n]))
+
+(defn- f-internal-view
+  []
+  (let [state       (reagent/atom :default)
+        active?     (atom false)
+        timer       (atom nil)
+        on-press-in (fn []
+                      (when-not (= @state :selected)
+                        (reset! timer (js/setTimeout #(reset! state :pressed) 100))))]
+    (fn [{:keys [contact-props accounts customization-color on-press theme]
+          :or   {customization-color :blue
+                 accounts            []}}]
+      (let [accounts-count (count accounts)
+            account-props  (when (= accounts-count 1)
+                             (first accounts))
+            on-press-out   (fn []
+                             (let [new-state (if @active? :default :active)]
+                               (when @timer (js/clearTimeout @timer))
+                               (reset! timer nil)
+                               (reset! active? (= new-state :active))
+                               (reset! state new-state)
+                               (when on-press
+                                 (on-press))))]
+        [rn/pressable
+         {:style               (style/container
+                                {:state @state :customization-color customization-color})
+          :on-press-in         on-press-in
+          :on-press-out        on-press-out
+          :accessibility-label :container}
+         [rn/view {:style style/left-container}
+          [user-avatar/user-avatar (assoc contact-props :size :small)]
+          [rn/view {:style style/saved-contact-container}
+           [rn/view
+            {:style style/account-title-container}
+            [text/text
+             {:weight :semi-bold
+              :size   :paragraph-1}
+             (:full-name contact-props)]
+            [rn/view
+             {:style               style/contact-icon-container
+              :accessibility-label :contact-icon}
+             [icon/icon :i/contact
+              {:no-color true
+               :size     12
+               :color    (colors/theme-colors
+                          (colors/custom-color :blue 50)
+                          (colors/custom-color :blue 60)
+                          theme)}]]]
```

Comment:
The `color` won't be applied if we pass true to the `no-color` prop.

P.S. If the icon has two colours, we need to use SVG. (ref `quo2.components.icons.svg` ns)

Reply:
True! Thanks for the suggestion, added the icon as SVG as I need to slightly change the backgrond color while preserving the white shapes inside